### DISCUSSION TITLE
Mason/lspconfig version of server

### DISCUSSION THREAD

**Discussion by dpetka2001** [LazyVim/LazyVim#144] [Q&A] [answered]
Do either Mason or nvim-lspconfig support some kind of `version` option for `ensure_installed` 
servers that I can define in their options? Something like:
```lua
opts = {
    ensure_installed = {
      "stylua",
      "shellcheck",
      {  "lua-language-server" , version = "x.y.z" },
    },
  },

**Comment by folke** [comment, marked as answer]
No idea. Instead of asking here, why not check the mason docs?

**Reply by dpetka2001** [reply]
Thank you for the advice. I checked in the discussions and it indeed does seem to support this.
For anyone else interested something like this would work:
```lua
require("mason-lspconfig").setup {
  ensure_installed = { "package@v1.0.0" }
}
```
